### PR TITLE
[Performance] Image Optimization <img> to <StaticImages> in the community page

### LINKED ISSUE #3016: [Performance] Image Optimization <img> to <StaticImages> in the community page
**Current Behavior**

The .png and .jpg images cause a lot of performance issues
**Desired Situation**

Change  to <StaticImagess> for image optimization in the handbook section. eg https://github.com/layer5io/layer5/pull/2597 [StaticImagePlugin](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/)
---
[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by tinniaru3005**
Signed-off-by: Arunima Chaudhuri <arunimachaudhuri2020@gmail.com>

**Description**
The .png and .jpg images cause a lot of performance issues
Image optimization in the handbook section.

This PR fixes #3016 

**Notes for Reviewers**
As the issue says to change only for png, and jpg formats so didn't change the SVGs. Also, changing the svgs was showing some errors.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8c1930efe34649e33ccd8b573bd90927f062859f at: https://63679af9aaf1f33a7ca11fd6--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 056c4445f4b145226a64ed2bf1ae713c1ea3bba0 at: https://6367b84545f9a948740a4585--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 52579adeaa0dcc85623f9586604349354cc38f90 at: https://6367bec8aaf1f35288a12336--layer5.netlify.app

File: src/sections/Community/Handbook/repository.js [hunk 1]
```diff
@@ -101,7 +102,7 @@ const Repository = () => {
 
               <li>
                 <a href="https://github.com/service-mesh-patterns">
-                  <img
+                  <StaticImage
                     src="https://user-images.githubusercontent.com/85789734/134711787-5bf0aeaa-008c-4e2c-a81e-e05f3dcfb54c.png"
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/repository.js:106]: Just use this [svg](https://github.com/layer5io/layer5/blob/master/src/assets/images/service-mesh-patterns/service-mesh-pattern.svg) here, that would be better. Reference the local path similar to other images above

**Comment by leecalcote** [comment]
@tinniaru3005 does this feedback make sense?

**Comment by tinniaru3005** [comment]
> @tinniaru3005 does this feedback make sense?

Hey @leecalcote, Ya it did. Gad my exams so couldn't update the changes. Will work on it today. Thanks for asking :)

**Comment by l5io** [comment]
🚀 Preview for commit 9f27cba106c60f60dce6bd10a06f489ac76f61af at: https://63733160cee60a0630d05f3c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f138f9783963a981716e0f2ea81ec99a7b4ff5de at: https://6373b636ee2a8c1c9772653a--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8e364e0c806e489c8fc9b30b8fe02185d3b15836 at: https://6373bd4d28519c23f0cac16f--layer5.netlify.app

File: src/sections/Community/Handbook/repository.js [hunk 2]
```diff
@@ -102,7 +103,7 @@ const Repository = () => {
               <li>
                 <a href="https://github.com/service-mesh-patterns">
                   <img
-                    src="https://user-images.githubusercontent.com/85789734/134711787-5bf0aeaa-008c-4e2c-a81e-e05f3dcfb54c.png"
+                    src={servicemeshpattern}
```
Comment (kayceeDev) [inline src/sections/Community/Handbook/repository.js:106]: looks good

**Review by kayceeDev — APPROVED** [review]
Going by the last feedbacks, you have adjusted `StaticImg` to `svg`. Good job.

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**